### DISCUSSION TITLE
React Hook Form showing react select field as undefined.

### DISCUSSION THREAD

**Discussion by inquisitive-stha** [react-hook-form/react-hook-form#7868] [Q&A] [answered]
Here is the [codepen link](https://codesandbox.io/s/react-hook-form-js-forked-sunwqb?file=/src/App.js)

1) Open the link 
2) Try to submit the form, on console, you can see `marketplaces` value as undefined. Although, the value on the form seems to be populated

If you change the value on that `select` field, and then submit, this time only the marketplaces values will be shown.

**Comment by Moshyfawn** [comment, marked as answer]
Followed your reproduction steps. Can't reproduce

<img width="546" alt="image" src="https://user-images.githubusercontent.com/16290753/154840613-962fae0f-6072-4d42-987a-a467da373336.png">

**Comment by inquisitive-stha** [comment]
Sorry, Actually, I had already fixed the issue, I was using `defaultValue` on select instead of `Controller`
